research/backend/manager/dbmanager.py

```diff
@@ -0,0 +1,6 @@
+import redis
+
+HOST = "localhost"
+DB = 0
+
+REDIS_DB = redis.StrictRedis(host=HOST, db=DB, charset="utf-8")
```

Comment:
Add `decode_responses=True` to your `StrictRedis` declaration parameters. It'll parse your Redis response to UTF-8 format.
Later, I'll provide a framework helps your code to easily interact with Redis.

Reply:
Okay, I added a parameter.

Reply:
Thanks for framework!!